crates/matrix-sdk/tests/integration/event_cache.rs

```diff
@@ -656,7 +660,7 @@ async fn test_backpaginating_without_token() {
     // If we try to back-paginate with a token, it will hit the end of the timeline
     // and give us the resulting event.
     let BackPaginationOutcome { events, reached_start } =
-        pagination.run_backwards(20).await.unwrap();
+        pagination.run_backwards(20, once).await.unwrap();
```

Comment:
Can you add tests where the `until()` method is *not* once?

Reply:
Done!